src/components/logic/calculate.js

```diff
@@ -0,0 +1,3 @@
+/* function (item) {
+   return !!item.match(/[0-9]+/);
+}; */
```

Comment:
- [ ] There are still commented codes as highlighted by the previous reviewers, please ensure they are removed